README.md

```diff
@@ -342,6 +344,58 @@ py_image(
 To use a Python 3 runtime instead of the default of Python 2, use `py3_image`,
 instead of `py_image`.  The other semantics are identical.
 
+### nodejs_image
+
+**It is notable that unlike the other image rules, `nodejs_image` is not
+currently using the `gcr.io/distroless/nodejs` image for a handful of reasons.**
+This is a switch we plan to make, when we can manage it.  We are currently
```

Comment:
I need to remove bash dependencies for a lighter windows install anyway, so we should follow up on this

Reply:
Is it possible to make the entrypoint itself Node.js?

If we can reduce the entrypoint logic to the point where I can just invoke Node directly (possibly bypassing a script that does that same) I'm happy to do that (I do that with Java).

Reply:
Yes, that is the plan